build-conflicts.php

```diff
@@ -232,16 +232,23 @@ function () use ($execute) {
     };
 
     $commitComposerJson = function (string $composerJsonPath) use ($runInPath, $execute) : void {
+        $originalCommit = $runInPath(
+            function () use ($execute) {
+               return $execute('git log -1 --oneline --no-merges');
```

Comment:
We ideally want to create a link such as `https://FriendsOfPhp/SecurityAdvisories/commit/aaaaaaaaaaabbbbbbbbbbccccccccccfddddffff`